errors/wait-until-non-streaming.md

```diff
@@ -0,0 +1,19 @@
+# The waitUntil API has limited support for non-streaming functions
+
+## Why This Warning Occurred
+
+The [waitUntil](https://vercel.com/docs/functions/functions-api-reference#waituntil) method allows you to enqueue asynchronous tasks to be performed during the lifecycle of the request, and these tasks can continue to resolve even after the response has been written.
+
+This is possible because `waitUntil` takes advantage of the [streaming functions](https://vercel.com/docs/functions/streaming#streaming-functions).
```

Comment:
```suggestion
This is possible because `waitUntil` takes advantage of [streaming functions](https://vercel.com/docs/functions/streaming#streaming-functions).
```